package.json

```diff
@@ -25,7 +25,7 @@
     "pretest": "npm run lint && npm run build",
     "test": "nyc npm run spec",
     "spec": "mocha --harmony --require esm script/setup.js --recursive test",
-    "prebuild": "rimraf dist",
+    "prebuild": "rm -r dist || true",
```

Comment:
Does this word on windows?

Reply:
Depends if they have WSL. Not likely. I'll revert since Windows users are a consideration